src/Evolution/Systems/GrMhd/ValenciaDivClean/PrimitiveFromConservative.cpp

```diff
@@ -178,9 +178,9 @@ bool PrimitiveFromConservative<OrderedListOfPrimitiveRecoverySchemes,
         equal_within_roundoff(
             total_energy_density[s],
             get(magnetic_field_squared)[s] * 0.5 + total_energy_density[s])) {
```

Comment:
So here is what happens.

At the line 88 of `Test_TciOnDgGrid.cpp`, magnetic field as a prim variable is set to be very very small (not sure why we chose to use a hydro regime case for testing the TCI?). So even if the following (line 319)
```
  const std::tuple<int, evolution::dg::subcell::RdmpTciData> result =
      db::mutate_apply<grmhd::ValenciaDivClean::subcell::TciOnDgGrid<
          grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin>>(
          make_not_null(&box), persson_exponent, element_stays_on_dg);
```
is executed with the `NewmanHamlin` scheme as the template parameter, due to this `equal_within_roundoff` consistency check that it'll run the `KastaunEtAlHydro` scheme.

Reply:
That's correct. KastaunHydro is now default regardless of the choice of MHD inversion.